src/templates/default/reviews/review_stages.html

```diff
@@ -0,0 +1,114 @@
+							<input name="proposals.creatable" type='hidden' value="false" />
+							<input id="proposals.creatable" name="proposals.creatable" type="checkbox" value="true"/>
+							<label for="proposals.creatable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.editable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.editable" type='hidden' value="false" />
+							<input id="proposals.editable" name="proposals.editable" type="checkbox" value="true"/>
+							<label for="proposals.editable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.withdrawable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.withdrawable" type='hidden' value="false" />
+							<input id="proposals.withdrawable" name="proposals.withdrawable" type="checkbox" value="true"/>
+							<label for="proposals.withdrawable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.visible.to.submitters"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="reviews.visible.to.submitters" type='hidden' value="false" />
+							<input id="reviews.visible.to.submitters" name="reviews.visible.to.submitters"
+								type="checkbox" value="true"/>
+							<label for="reviews.visible.to.submitters" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.stage"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input id="reviews.stage" name="reviews.stage" type="number" min="0" max="2"
+								placeholder="Int" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.disable.after"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input name="proposals.disable.after" type="datetime-local" step="1"
+								placeholder="yyyy-mm-dd hh:mm:ss" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						timezone
+						<div class="input-customized-size pull-right">
+							<select name="review_timezone">
+								{% for tz in timezones %}
+								<option value="{{ tz }}" {% if tz == TIME_ZONE %} selected{% endif %}>{{ tz }}</option>
+								{% endfor %}
+							</select>
+						</div>
+					</li>
+				</ul>
+			</div>
+
+			<div class="nesting-form-group">
+				{{ form|crispy }}
+				<button type="submit" class="btn btn-info pull-right">
+					{% trans 'Submit' %}
+				</button>
+			</div>
+		</form>
+	</div>
+</div>
+
+{% endblock main-content %}
+
+{% block extra_js %}
+{% compress js %}
+<script src="{% static 'js/reviews/review_stages.js' %}"></script>
+{% endcompress %}
+{% endblock extra_js %}
```

Comment:
I think you should read current values from the database and show current settings on page load.

Something like
```html
<script src="{% static 'js/reviews/review_stages.js' %}"></script>
<script>
proposals_creatable.checked = {% if proposals_creatable %}true{% else %}false;
proposals_editable.checked = {% if proposals_editable %}true{% else %}false;
proposals_withdrawable.checked = {% if proposals_withdrawable %}true{% else %}false;
reviews_stage.value = {{ reviews_stage }};
reviews_visible_to_submitters.checked = {% if reviews_visible_to_submitters %}true{% else %}false;
</script>
```

The above is just a sample code, very ugly, but just a simple demonstration of how django template variables and javscript can be integrated together.

Reply:
There are currently two ways to display the current settings. The first is that your suggestions are automatically loaded into the form that will be submitted (500b9db18c69e3283379343807f3ad5d06a57ad7). The second is to list the current settings in another list (2fb312d1dfc1c626037d9dbebae6f53e2d438a2b)

Reply:
Maybe the two functions of the current review stage status and the set review stage can be presented in different pages ？Do you have more ideas ?

Reply:
uh... what I mean is that the boolean switches should represent the current set values. So that when the admin sees the page, current settings are clear. And the admin can do some minor tuning without "not knowning" what current values look like.

我想用中文可能比較好表達。原本是不會載入目前的設定的，所以每次進來這一頁都是 false、或者是空的值。這樣其實很難一目瞭然目前到底是哪一個階段，或者是說目前的設定到底是長怎樣。期待的樣子是我進來的時候，所有的值就是目前的設定。舉個很爛的例子，假設目前是 editable、withdrawable 是開的，然後突然要很臨時的開放 creatable 一小段時間，我就只要把 creatable 打開，然後時間過了我再把它關起來。我就不需要再一個一個確認其他某一個的值是什麼，然後小心翼翼的調整怕弄錯，然後改完又要再小心翼翼的改回去。